lib/jquery-plugin/jquery.vscrlist.js

```diff
@@ -372,10 +372,12 @@ vscrlist.prototype.listTopIndex = function(index) {
  * @returns {boolean} status whether the index was updated.
  */
 vscrlist.prototype._listTopIndex = function(index) {
-    if(index < 0) {
+    const indexMax = (this._items.length - this._visible_row_count) ? 0:
```

Comment:
The condition incorrect.
It should be `this._items.length - this._visible_row_count < 0`.